From ea605b660e04878836cbae9f7d3dc0b4a9c893a6 Mon Sep 17 00:00:00 2001
From: Aeon <171243930+aeon1911@users.noreply.github.com>
Date: Mon, 28 Sep 2026 12:29:43 -0300
Subject: [PATCH 08/56] Show uz5.0.3-general on the launcher screen.

The corner label still said nx3.7 after the nacp version had already moved.
---
 tools/nxlauncher/src/platform.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/nxlauncher/src/platform.h b/tools/nxlauncher/src/platform.h
index 48bc49c14..cf1a25edd 100644
--- a/tools/nxlauncher/src/platform.h
+++ b/tools/nxlauncher/src/platform.h
@@ -8,7 +8,7 @@
 #define TMPDIR BASEDIR "/temp"
 
 #undef VERSION
-#define VERSION "uz5.0.3-nx3.7"
+#define VERSION "uz5.0.3-general"
 
 enum Button
 {
-- 
2.56.0

